R/rdataretriever.R

```diff
@@ -88,22 +88,40 @@ install = function(dataset, connection, db_file=NULL, conn_file=NULL,
 #' }
 fetch = function(dataset, quiet=TRUE){
   temp_path = tempdir()
-  if (quiet)
+  master = vector("list", length(dataset))
+  names(master) = dataset
+  names(master) = gsub("-","_",names(master))
+  z = 1
```

Comment:
Thank you for the contribution @maxpohlman. It would be nice to have the variables name reflecting what they store for easy readability.